### PR TITLE
add ccache support in docker images

### PR DESCRIPTION
**Pull request by AmeBel**

### CODE REVIEW

File: opencog/tools/ide/Dockerfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ MAINTAINER Alex van der Peet "alex.van.der.peet@gmail.com"
 MAINTAINER David Hart "dhart@opencog.org"
 
 # Install non-system project dependencies
-RUN  /tmp/octool -al
+RUN  /tmp/octool -al ; ccache -C
```
Comment (wenwei-dev) [inline opencog/tools/ide/Dockerfile]: I think the ccache should not be cleared. We should reuse the persisted ccache.

Comment (AmeBel) [inline opencog/tools/ide/Dockerfile]: we don't need the cache as part of the image, it will further blot the image.

Comment (AmeBel) [inline opencog/tools/ide/Dockerfile]: For run you can use mount a directory. That is what is done by docker-compose.
```
mkdir .ccache
docker run -v .ccache:/home/opencog/.ccache ...
```

Comment (wenwei-dev) [inline opencog/tools/ide/Dockerfile]: I see. It's the cache created when building the docker image. Yeah we don't need it.

**Pull request merged**